vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -880,6 +881,14 @@ public void deletePictureResource(String uri, ModelCallback callback) {
         return call;
     }
 
+    @Nullable
+    public retrofit2.Response<Video> fetchVideoSync(String uri) {
```

Comment:
I thought about it, but then returning "null" in the case of a failure isn't really descriptive enough - the caller may want to know if it was for network or what the the status code was.